src/main/java/teammates/storage/api/StudentsDb.java

```diff
@@ -336,14 +331,6 @@ private void updateStudentDetails(String newName, String newTeamName, String new
      *
      */
 
-    public void deleteStudent(String courseId, String email) {
-        deleteStudent(courseId, email, true);
-    }
-
-    public void deleteStudentWithoutDocument(String courseId, String email) {
-        deleteStudent(courseId, email, false);
-    }
-
     public void deleteStudent(String courseId, String email, boolean hasDocument) {
```

Comment:
The parameter `boolean hasDocument` should be removed instead.

I think it is OK to delete document even though it might not exist (although it should exist for each student).

Reply:
removed the boolean parameter from all the methods which delete students. since document exists for each student, can I remove the boolean parameter from `create & update` student methods as well.

Reply:
Yes. That is what the issue about.

Reply:
I removed hasDocument param from creating and updating methods as well.